include/SDL3/SDL_surface.h

```diff
@@ -64,6 +64,17 @@ extern "C" {
 
 typedef struct SDL_BlitMap SDL_BlitMap;  /* this is an opaque type. */
 
+/**
+ * The scaling mode
+ */
+typedef enum
+{
+    SDL_SCALEMODE_NEAREST, /**< nearest pixel sampling */
+    SDL_SCALEMODE_LINEAR,  /**< linear filtering */
+    SDL_SCALEMODE_BEST     /**< anisotropic filtering */
+} SDL_ScaleMode;
```

Comment:
Maybe we should have a new header that can be shared, like we did for SDL_blendmode.h?

Reply:
yes, I saw that, but that's only 1 struct. blendmode have more data.
this could be move to SDL_video.h, pixel.h, ...

( maybe we should remove SDL_SCALEMODE_BEST which is unused currently.)